tag-pose/aruco_pose.py

```
@@ -0,0 +1,182 @@
+            for marker_id, corners in zip(ids, corners_list):
+                image_points = corners.reshape(4, 2).astype(np.float32)
+                success, rvec, tvec = cv2.solvePnP(
+                    object_points,
+                    image_points,
+                    camera_matrix,
+                    distortion_coeffs,
+                    flags=cv2.SOLVEPNP_IPPE_SQUARE,
+                )
+
+                contour = image_points.astype(int).reshape((-1, 1, 2))
+                cv2.polylines(display, [contour], True, color, 2, cv2.LINE_AA)
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)